packages/compass-query-bar/src/components/query-bar/query-bar.spec.js

```diff
@@ -242,5 +242,91 @@ describe('QueryBar [Component]', function() {
         });
       });
     });
+
+    describe('when rendered with or without a query history button', function() {
+      const layout = ['filter'];
+
+      it('query history button renderes by default', function() {
+        const component = shallow(
+          <QueryBar
+            store={store}
+            actions={actions}
+            layout={layout}
+            expanded
+            serverVersion="3.4.0" />
+        );
+        expect(component.find('button[data-test-id="query-history-button"]')).to.exist;
+      });
+
+
+      it('query history button renderes when showQueryHistoryButton prop is passed and set to true', function() {
+        const component = shallow(
+          <QueryBar
+            store={store}
+            actions={actions}
+            layout={layout}
+            showQueryHistoryButton
```

Comment:
If this is `false` by default do we also need to enable it where is used in Compass?

Reply:
I think it's `true` by default (see previous test and defaultProps in the component), so this test is just a bit redundant (also a bit confusing because of that 😅 )

Reply:
@mcasimir Sorry I missed your comment earlier. `showQueryHistoryButton` and `showExportToLanguageButton` props are true by default as @gribnoysup commented. I did that so that I would not change the appearance of the compass query bar out of the box, but that we would also have the option to disable it when we want to. The redundant test you are referring to is present just make sure that those buttons are present when these two props are not explicitly set. I am happy to remove them if you think that would be better.

Reply:
I think even thought it's a bit redundant, it's okay to keep it there 😊